test/e2e/hermeticmode/hermetic_test.go

```diff
@@ -0,0 +1,390 @@
+package hermeticmodetest
+
+import (
+	"context"
+	"os"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
```

Comment:
Could be helpful to add a comment here to explain how those test are working. Took me a while to figure out who is writing to EnvOutput.